nixfmt-bot.nix

```diff
@@ -2,19 +2,19 @@ let
   buildDepError = pkg:
     builtins.throw ''
       The Haskell package set does not contain the package: ${pkg} (build dependency).
-
+      
```

Comment:
Please configure your editor to delete trailing whitespace :wink:

Reply:
That's an autogenerated file :)